src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -466,32 +466,68 @@ private function registerWorkflowConfiguration(array $config, ContainerBuilder $
         foreach ($config['workflows'] as $name => $workflow) {
             $type = $workflow['type'];
 
+            // Process Metadata (workflow + places (transition is done in the "create transition" block))
+            $metadataStoreDefinition = new Definition(Workflow\Metadata\InMemoryMetadataStore::class, array(null, null, null));
```

Comment:
We tend to always use `use` statements.

Reply:
Actually for the workflow I only mounted the Workflow namespace. It is cleaner IMHO